ethstorage/archiver/guide-devnet.md

```diff
@@ -342,17 +291,81 @@ make op-node
   --rpc.enable-admin \
   --l1=$L1_RPC_URL \
   --l1.rpckind=$L1_RPC_KIND \
-  --l1.beacon=http://localhost:3602 \
-  --l1.beacon-archiver http://localhost:6678
+  --l1.beacon=http://localhost:3602 
 ```
 
 **Note:**
 1. P2P is disabled so that it can only sync data from L1.
 2. The L1 beacon URL is directed to the beacon proxy, where blobs expire quickly.
-3. The beacon archiver is configured to point to the es-node archive service. 
 
-## Verifying Sync Status
+You can observe that the validator node is unable to sync properly with "derivation failed" error due to "failed to fetch blobs". 
+
+Now, navigate to the parent directory in preparation for the next steps.
+
+## Running EthStorage Node
+
+### Installation
+
+To set up the es-node, first clone the repository and build it:
+```bash
+git clone https://github.com/ethstorage/es-node.git
+cd es-node
+git checkout v0.1.16
+make
+```
+
+### Initialization
+
+Initialize es-node:
+```bash
+./init-rpc.sh \
+--l1.rpc http://localhost:8545 \
+--storage.l1contract $ES_CONTRACT
+```
+
+### Running ES Node in Archiver Mode
 
+Retrieve the beacon genesis time for later use:
+```bash
+curl -s http://localhost:5052/eth/v1/beacon/genesis | jq -r '.data.genesis_time'
+
+1732529739
+
+export GENESIS_TIME=1732529739 // replace with the actual timestamp
+```
+Note: Before proceeding to the next step of launching the es-node, ensure that at least 2 epochs (approximately 13 minutes) have passed since the EthStorage contracts were deployed in [this step](#deploying-ethstorage-contracts), as the es-node needs to read the finalized states of the contract.
+
+Finally, run the es-node:
```

Comment:
Should we mention that at least 2 epochs need to pass after deploying the storage contract before launching the es-node? Qi encountered this issue while following the documentation.